docs/src/_getting-started/part4.md

```diff
@@ -1,171 +0,0 @@
----
-title: Part 4 - Inline Data Mapping
```

Comment:
@sbalnojan I like most of the PR except for this section. Mappers are still very relevant for this use case and fit a software engineering ICP. I would ask that we add this back but instead of hashing PII maybe we filter or duplicate some data?

Reply:
@tayloramurphy, I think I disagree. For software engineers, this is a nice-to-have (and totally not needed) feature, whereas, for a data engineer, it is a must-have. 


My plan would be to:
1. Extract the complete "internal data team" tutorial, where the inline mapper would sit as well
2. And have the introductory tutorial without it.